src/core/webidl.js

```diff
@@ -49,9 +51,22 @@ const templates = {
       case "object":
         type = "interface";
         cite = "WebIDL";
+        break;
+      default: {
+        const isWorkerType = unescaped.includes("Worker");
+        if (isWorkerType && tokens && tokens.name.value === "Exposed") {
+          lt = `${unescaped}GlobalScope`;
+          type = "interface";
+          cite = ["Worker", "DedicatedWorker", "SharedWorker"].includes(
```

Comment:
Is this some optimization thing?

Reply:
No, cite needs to be explicit for now or it breaks.